specification/wse/src/main/scripts/org/kaazing/specification/wse/opening/request.header.x.websocket.protocol/handshake.request.rpt

```diff
@@ -33,6 +33,6 @@ read version "HTTP/1.1"
 read header "X-WebSocket-Protocol" /primary|secondary/
 read header "Content-Type" /text\/plain;charset=(?i)utf-8/
 
-read "http://localhost:8080/path/uofdbnreiodfkbqi\n"
-read "https://localhost:8443/path/kwebfbkjwehkdsfa\n"
+read /http:\/\/localhost:8080\/path\/.+?\n/
+read /http:\/\/localhost:8080\/path\/.+?\n/
```

Comment:
Perhaps it would be slightly more readable to use the following syntax instead?
`read "http://localhost:8080/path/" /.+?\n/`

Also, what did you mean by `.+?\n`? Is that different from `.*\n`?